src/components/Reports/GoalCalculator/SummaryReport/MpdGoal/MpdGoalTable.tsx

```diff
@@ -0,0 +1,361 @@
+      line: '2',
+      category: t('Benefits Charge'),
+      amount: ministryExpenses.benefitsCharge,
+      reference: 2302.24,
+    },
+    {
+      line: '3',
+      category: t('Ministry Mileage'),
+      amount: ministryExpenses.ministryMileage,
+      reference: 95,
+    },
+    {
+      line: '4',
+      category: t('Medical Mileage'),
+      amount: ministryExpenses.medicalMileage,
+      reference: 65,
+    },
+    {
+      line: '5',
+      category: t('Medical Expenses'),
+      amount: ministryExpenses.medicalExpenses,
+      reference: 230,
+    },
+    {
+      line: '6',
+      category: t('Ministry Partner Development'),
+      amount: ministryExpenses.ministryPartnerDevelopment,
+      reference: 155,
+    },
+    {
+      line: '7',
+      category: t('Communications'),
+      amount: ministryExpenses.communications,
+      reference: 135,
+    },
+    {
+      line: '8',
+      category: t('Entertainment'),
+      amount: ministryExpenses.entertainment,
+      reference: 125,
+    },
+    {
+      line: '9',
+      category: t('Staff Development'),
+      amount: ministryExpenses.staffDevelopment,
+      reference: 180,
+    },
+    {
+      line: '10',
+      category: t('Supplies'),
+      amount: ministryExpenses.supplies,
+      reference: 50,
+    },
+    {
+      line: '11',
+      category: t('Technology'),
+      amount: ministryExpenses.technology,
+      reference: 100,
+    },
+    {
+      line: '12',
+      category: t('Travel'),
+      amount: ministryExpenses.travel,
+      reference: 225,
+    },
+    {
+      line: '13',
+      category: t('Transfers'),
+      amount: ministryExpenses.transfers,
+      reference: 160,
+    },
+    {
+      line: '14',
+      category: t('Other'),
+      amount: ministryExpenses.other,
+      reference: 80,
+    },
+    {
+      line: '15',
+      category: t('Ministry Expenses Subtotal'),
+      amount: calculations.totalMinistryExpenses,
+      reference: 2994.74,
+    },
+    {
+      line: '16',
+      category: t('Subtotal'),
+      amount: calculations.overallSubtotal,
+      reference: 10224.63,
+    },
+    {
+      line: '17',
+      category: t('Subtotal with 12% admin charge'),
+      amount: calculations.overallSubtotalWithAdmin,
+      reference: 11618.9,
+    },
+    {
+      line: '18',
+      category: t('Total Goal (line 16 x 1.06 attrition)'),
+      amount: calculations.overallTotal,
+      reference: 12316.03,
+    },
+    {
+      line: '19',
+      category: t('Solid Monthly Support Developed'),
+      amount: calculations.supportRaised,
+      reference: 0,
+    },
+    {
+      line: '20',
+      category: t('Monthly Support to be Developed'),
+      amount: calculations.supportRemaining,
+      reference: 12316.03,
+    },
+    {
+      line: '21',
+      category: t('Support Goal Percentage Progress'),
+      amount: calculations.supportRaisedPercentage,
+      reference: 0,
+      percentage: true,
+    },
+  ];
+
+  const columns: GridColDef[] = [
```

Comment:
`valueFormatter` and `columns` could be defined outside the component so they don't need to be re-created

Reply:
`valueFormatter` depends on `t` and `locale`, and `columns` depends on `t`. It didn't seem worth pulling those definitions out and passing in `t` and `locale`.